File: Java/SCRUM.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Main.java to edit this template
 */
package scrum;

/**
 *
 * @author galindo
 */
public class SCRUM {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        Proyecto proyecto = new Proyecto("Proyecto 1", "20-19-2023", true);
        
        Stakeholder guillermo = new Stakeholder(100, "memo", 0);
        proyecto.stakeholders.add(guillermo);
        
        ScrumMaster pepe = new ScrumMaster("pepe");
        Equipo equipo = new Equipo("Bum Bum", 0, pepe);
        
        Integrante juan = new Integrante("Juan", "Diseñador");
        equipo.integrantes.add(juan);
        
        Integrante leo = new Integrante("Leo", "Analista");
        equipo.integrantes.add(leo);
        
        proyecto.equipos.add(equipo);
        
        System.out.println(proyecto);
    }
    
}

File: Python/SCRUM.py
from Proyecto import Proyecto
from Stakeholder import Stakeholder
from ScrumMaster import ScrumMaster
from Equipo import Equipo
from Integrante import Integrante

proyecto = Proyecto("Proyecto 1", "20-19-2023", True)

guillermo = Stakeholder(100, "memo", 0)
proyecto.stakeholders.append(guillermo)

pepe = ScrumMaster("pepe")
equipo = Equipo("Bum Bum", 0, pepe)

juan = Integrante("Juan", "Diseñador")
equipo.integrantes.append(juan)

leo = Integrante("Leo", "Analista")
equipo.integrantes.append(leo)

proyecto.equipos.append(equipo)

print(proyecto)